vogue/build/lims_utils.py

```diff
@@ -10,15 +10,14 @@ def str_to_datetime(date: str)-> dt:
         return None
     return dt.strptime(date, '%Y-%m-%d')
 
+
 def get_sequenced_date(sample: Sample, lims: Lims)-> dt:
     """Get the date when a sample passed sequencing."""
 
-    #process_type = 'CG002 - Sequence Aggregation' unsure wich we should use acually
     process_types = ['CG002 - Illumina Sequencing (HiSeq X)', 
-                     'CG002 - Illumina Sequencing (Illumina SBS)'] 
+                     'CG002 - Illumina Sequencing (Illumina SBS)']  #process_type = 'CG002 - Sequence Aggregation' unsure wich we should use acually
```

Comment:
```suggestion
                     #process_type = 'CG002 - Sequence Aggregation' unsure which we should use actually
                     'CG002 - Illumina Sequencing (Illumina SBS)']  
```
We should not have lines longer than 100